### ISSUE TITLE
Key-Service with Inputstream 2.2.16

### ISSUE THREAD

**Issue by DiR3Wolf** [eracknaphobia/plugin.video.psvue#23]
@eracknaphobia or @snoopyjoe  can either one of you test PSVue with the latest Kodi 18 nightly that includes inputstream 2.2.16?

After installing this version, I am able to browse PS Vue, but I am not able to play any channel. I get the following error:  https://pastebin.com/Cvc1pUi7

It looks like "Key" certificate is failing. I don't know if it is changes on inputstream, or it is somehow my account. Everything works when I run inputstream 2.2.14 , So it either broke on .15 or .16. Thanks!!!

Edit: I just checked my other computer with inputstream 2.2.15, problem seems to be introduced on 2.2.16

**Comment by snoopyjoe** [comment]
Wow never seen that error before, but it definitely something changed in Inputstream. I don't think issue exist in 2.2.15 as I'm currently using that version but I'm also not running the latest nightly. If I get my hands on a compiled 2.2.16 for my boxes I'll test it.

**Comment by DiR3Wolf** [comment]
Yes, the last working Kodi version, at least for me, is 04-17-18. As of yesterday, 4/19 and 4/20 nightly weren't working. The relevant changes since 4-17-18 are on inputstream and FFMPEG updated to 4.0. I don't know which change is the one giving me issues.

**Comment by DiR3Wolf** [comment]
Weird. I just reinstalled the same nightly that wasn't working yesterday. I thought about changing the binary folder for inputstream to see if I could make it work and narrow down the issue, but all of the sudden it started working. Another thing that I did was installing widevine as drm, (because inputstream looks for it) but I don't think it has anything to do with PS Vue, I'm sure they don't use that. So, maybe it was my account for some idiotic reason. I'm closing the issue for now.
